frontend/server/src/Controllers/Problem.php

```diff
@@ -3733,13 +3734,17 @@ public static function apiList(\OmegaUp\Request $r) {
         $rowcount = \OmegaUp\Controllers\Problem::PAGE_SIZE;
 
         $onlyQualitySeal = $r->ensureOptionalBool('only_quality_seal') ?? false;
+        $level = null;
```

Comment:
```suggestion
        $level = $r->ensureOptionalInt('level');
```

hay que intentar no usar `$r[...]` jamás.